src/font.c

```diff
@@ -1114,6 +1120,13 @@ gdip_create_font_from_logfont (void *hdc, void *lf, GpFont **font, BOOL ucs2)
 		result->face[LF_FACESIZE - 1] = '\0';
 	}
 
+#ifdef USE_PANGO_RENDERING
+	result->pango = NULL;
+#else
+	result->cairofnt = NULL;
+	result->cairo = NULL;
+#endif
+
```

Comment:
Without this change, we actually hit an assertion when deleting the font from inside cairo